core/python_utils_test.py

```diff
@@ -54,7 +54,7 @@ def test_open_file(self):
 
     def test_can_not_open_file(self):
         with self.assertRaisesRegex(
-            IOError, 'Unable to open file: invalid_file.py'):
+            FileNotFoundError, 'Unable to open file: invalid_file.py'):
```

Comment:
```suggestion
            FileNotFoundError, 'Unable to open file: invalid_file.py'
        ):
```

Reply:
@vojtechjelinek i have ported over the tests to utils_test.py as it's entirely python3 supporting now. python_utils_test.py does not contain any test code for open_file function now . I believe you are looking at some older commits .